File: Problems/ICPC Asyut Sheets/Sheet #2 (Loops)/C. Even, Odd, Positive and Negative.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int x;
    int even = 0, odd = 0, pos = 0, neg = 0;
    while (n--)
    {
        cin >> x;
        (x % 2) ? odd++ : even++;
        if (x > 0)
            pos++;
        else if (x < 0)
            neg++;
    }
    cout << "Even: " << even << "\n"
         << "Odd: " << odd << "\n"
         << "Positive: " << pos << "\n"
         << "Negative: " << neg;

    return 0;
}

File: Problems/ICPC Asyut Sheets/Sheet #2 (Loops)/C. Even, Odd, Positive and Negative.py
n = int(input())
arr = list(map(int, input().split()))

pos, neg, even, odd = 0, 0, 0, 0

for x in arr:
    if x % 2 == 0:
        even += 1
    else:
        odd += 1

    if x > 0:
        pos += 1
    elif x < 0:
        neg += 1

print(f"Even: {even}\nOdd: {odd}\nPositive: {pos}\nNegative: {neg}")
